src/solc_0.8/catalyst/GemsCatalystsRegistry.sol

```diff
@@ -244,11 +247,11 @@ contract GemsCatalystsRegistry is WithSuperOperators, ERC2771Handler, IGemsCatal
         emit TrustedForwarderChanged(trustedForwarder);
     }
 
-    function _msgSender() internal view override(Context, ERC2771Handler) returns (address sender) {
+    function _msgSender() internal view override(ContextUpgradeable, ERC2771Handler) returns (address sender) {
         return ERC2771Handler._msgSender();
     }
 
-    function _msgData() internal view override(Context, ERC2771Handler) returns (bytes calldata) {
+    function _msgData() internal view override(ContextUpgradeable, ERC2771Handler) returns (bytes calldata) {
         return ERC2771Handler._msgData();
     }
 }
```

Comment:
can you fix the name of `setGemsandCatalystsMaxAllowance`
-> `setGemsAndCatalystsMaxAllowance`